docs/upgrade/upgrade-2.*.md

```diff
@@ -5,6 +5,11 @@ nav_order: 0
 parent: Upgrade
 ---
 
+# Upgrade from `v2.4.0` to `v2.4.1`
```

Comment:
To me this is not a bugfix but an added feature so it should be released as tag 2.5.0...

Reply:
Yep, I was very undecided too. When in doubt, it's better to opt for a new minor!